OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameCashCounterLogic.cs

```diff
@@ -18,12 +19,12 @@ public class IngameCashCounterLogic : ChromeLogic
 		[ObjectCreator.UseCtor]
 		public IngameCashCounterLogic(Widget widget, World world)
 		{
-			var playerResources = world.LocalPlayer.PlayerActor.Trait<PlayerResources>();
+			var resourceDisplay = world.LocalPlayer.PlayerActor.TraitsImplementing<IResourceDisplay>().Single();
```

Comment:
`Trait<T>` will do the same thing as `TraitsImplementing<T>().Single()` but with more specific errors, so I would stick with that.